napari/_qt/qt_main_window.py

```diff
@@ -616,9 +622,9 @@ def _add_menus(self):
         self.main_menu.addMenu(self.window_menu)
         self.plugins_menu = menus.PluginsMenu(self)
         self.main_menu.addMenu(self.plugins_menu)
-        self.help_menu = menus.HelpMenu(self)
-        self.main_menu.addMenu(self.help_menu)
```

Comment:
you still need this line with `addMenu` ... that's why the help menu isn't showing up

Reply:
nice catch! thanks! accidentally dropped 2 lines instead of one 😅